config/v1/types_console.go

```diff
@@ -20,7 +20,7 @@ type Console struct {
 
 type ConsoleSpec struct {
 	// +optional
-	Authentication ConsoleAuthentication `json:"authentication,omitempty"`
```

Comment:
@benjaminapetersen these changes here in `types_console.go` do not change serialization behaviour. But please double check you really meant what you specified with tags (and fix in a follow-up if you haven't).

In this case here: that authentication field is no pointer and will never be omited in JSON marshalling.

Reply:
The current behavior works for me, I have noticed that `authentication` is generated as an empty struct.

Reply:
@spadgett fyi